app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="each_post">
+    <li>
+        <div class="user-posts">
+           <%= image_tag post.creator.gravatar_url %> <span class="mr-3  name1"> 
+           <%= post.creator.name.capitalize %> <span> 
```

Comment:
Let's not forget that `capitalize` only capitalizes the first letter of a string:

```
irb(main):002:0> 'john doe'.capitalize
=> "John doe"
irb(main):003:0> 
```

I'm sure this isn't what you want:

![image](https://user-images.githubusercontent.com/22228397/65265590-e7e65800-dad6-11e9-9180-0307f8bc9bb4.png)


[`OPTIONAL`]